Posts/2021/02/Get-Services, PowerShell 7 VS Windows PowerShell 5.1

```diff
@@ -0,0 +1,150 @@
+---
+post_title: Get-Services, PowerShell 7 VS Windows PowerShell 5.1
```

Comment:
```suggestion
post_title: Using Get-Service in PowerShell 7 vs. Windows PowerShell 5.1
```

Reply:
Got it.
Sorry for my English, it the second lang :)